usaspending_api/reporting/v2/views/agencies/agency_code/overview.py

```diff
@@ -99,6 +110,10 @@ def format_results(self, result_list):
                 "fiscal_year": result["fiscal_year"],
                 "fiscal_period": result["fiscal_period"],
                 "current_total_budget_authority_amount": result["total_budgetary_resources"],
+                "total_budgetary_resources": result["gtas_total_budgetary_resources"],
+                "percent_of_total_budgetary_resources": result["total_budgetary_resources"]
+                * 100
+                / result["gtas_total_budgetary_resources"],
```

Comment:
I assume the results of this equation will maintain a strict two decimal places from the Decimals returned by the queryset. But I would prefer not to assume. Can you confirm?

Reply:
I can confirm it does not. I will make it so.

Reply:
rounded (and updated contract doc)